rules/rules-reviewed/eap8/eap7/jboss-dependencies.windup.xml

```diff
@@ -376,19 +403,7 @@
                 <hint title="Replace org.hibernate group with org.hibernate.orm" effort="1" category-id="mandatory">
                     <message>Update the dependency by replacing the group `org.hibernate` artifact `hibernate-jpamodelgen`  
                         with group `org.hibernate.orm` artifact `hibernate-jpamodelgen`</message>
-                </hint>
-            </perform>
-        </rule>
-        <rule id="jboss-dependencies-00029">
-            <when>
-                <project>
-                    <artifact groupId="org.jboss.narayana.xts" artifactId="jbossxts"/>
-                </project>
-            </when>
-            <perform>
-                <hint title="Replace jbossxts artifact with jbossxts-jakarta" effort="1" category-id="mandatory">
-                    <message>Update the dependency by replacing the group `org.jboss.narayana.xts` artifact `jbossxts`  
-                        with group `org.jboss.narayana.xts` artifact `jbossxts-jakarta`</message>
```

Comment:
Hi @emmartins - can I ask why this rule has been removed? I looked at the new linked article https://access.redhat.com/articles/6980017 and that seems to me to say that ```replacing the group `org.jboss.narayana.xts` artifact `jbossxts`  
                        with group `org.jboss.narayana.xts` artifact `jbossxts-jakarta``` is still valid. The very last line of the article references this change